packages/preset-icons/src/types.ts

```diff
@@ -52,7 +54,7 @@ export interface IconsOptions {
   /**
    * Custom icon customizations.
    */
-  customizations?: Omit<IconCustomizations, 'additionalProps' | 'trimCustomSvg'>
+  customizations?: Omit<IconCustomizations, 'additionalProps' | 'trimCustomSvg' | 'iconCustomizer'>
```

Comment:
remove this, the type above, also `iconCustomizer` entry below, it must be done inside `customizations` option